people-ops/international-compensation.html.md

```diff
@@ -0,0 +1,75 @@
+---
+title: International compensation
+meta_description: >
+  Nebulab strives to hire the best fit-talents both in culture and skill-set. Our international
+  compensation policy aims to pay the right salary for them. Find out how we handle this!
+---
+
+> Note: The work here is inspired by [Gitlab's approach to international compensation](https://about.gitlab.com/handbook/total-rewards/compensation/#paying-local-rates),
+> which we believed to be a good starting point.
+
+We hire people in different regions and countries. Every market is different, and it can be
+challenging to be competitive and fair towards other team members in making an offer for the same
+role but in a different region. The best way to deal with that is by introducing a Location factor.
+
+### Why we pay local salaries
+
+The key argumentations on why we value this policy as a critical success factor, both in hiring
+talented people and being fair and profitable as a company, are:
+
+- having a strict salary structure will bring low competitiveness in high salary regions – this
+mainly means that we can't hire people in that area;
+- having a strict salary structure will bring high competitiveness in low salary regions – this will
+bring a geographical concentration of team members, instead of a distributed team which represents
+our main goal. Besides, people who are unhappy with their job are less likely to leave if their
+compensation is higher than the average market one;
+- it's fair for anyone having the same wage for the same role, as far as discretional – team members
+are treated equally by the company, which brings a sense of fairness to the team.
+
+## The Location factor
+
+The Location factor is a multiplier of our former salary matrix.
+
+An important note is that we determine factors by using each country's labor index cost instead of
+the cost of living. The difference between them is thin but still relevant:
+
+- cost of labor is the cost to hire and retain local nationals;
+- cost of living is the cost to maintain a certain standard of living within a geographic location.
+
+It goes without saying that the latter can be far higher than the first one; in some cases, it will
+result in paying someone far more than what he's going to obtain from his local market just because
+he lives in an expensive area.
```

Comment:
Maybe an example here could help to clarify the difference between these two.

Reply:
I'm gonna try adding one.